pkgs/development/python-modules/ucsmsdk/default.nix

```diff
@@ -0,0 +1,38 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+, pyparsing
+, setuptools
+, six
+}:
+
+buildPythonPackage rec {
+  pname = "ucsmsdk";
+  version = "0.9.14";
+  format = "setuptools";
+
+  src = fetchFromGitHub {
+    owner = "CiscoUcs";
+    repo = "ucsmsdk";
+    rev = "v${version}";
+    sha256 = "sha256-lSkURvKRgW+qV1A8OT4WYsMGlxxIqaFnxQ3Rnlixdw0=";
+  };
+
+  propagatedBuildInputs = [
+    pyparsing
+    setuptools
```

Comment:
Even though it's listed in `requirements.txt`, I don't think this is a necessary propagated dependency. Maybe we can remove it using `pythonRelaxDepsHook`.

Reply:
It isn't even reading that file and it just works without it.